llvm/lib/Target/AArch64/AArch64FrameLowering.cpp

```diff
@@ -1214,8 +1214,7 @@ static MachineBasicBlock::iterator convertCalleeSaveRestoreToSPPrePostIncDec(
       SEH->eraseFromParent();
   }
 
-  TypeSize Scale = TypeSize::getFixed(1);
-  unsigned Width;
+  TypeSize Scale = TypeSize::getFixed(1), Width(0, false);
```

Comment:
I think it would be nice to use consistent notation here and have 

 TypeSize Scale = TypeSize::getFixed(1), Width = TypeSize::getFixed(0);